examples/flutter_gallery/lib/demo/color_testing_demo.dart

```diff
@@ -0,0 +1,89 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+
+class ColorTestingDemo extends StatelessWidget {
+  const ColorTestingDemo({ Key key }) : super(key: key);
+
+  static const String routeName = '/color_demo';
+
+  @override
+  Widget build(BuildContext context) => new ColorDemoHome();
+}
+
+const double _kPageMaxWidth = 500.0;
+
+class ColorDemoHome extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    final double statusBarHeight = MediaQuery.of(context).padding.top;
+    return new Scaffold(
+      appBar: new AppBar(title: const Text('Color Demo')),
+      body: new ListView(
+        padding: new EdgeInsets.all(5.0),
+        children: <Widget>[
+          new Image.asset('packages/flutter_gallery_assets/ali_connors.jpg', fit: BoxFit.contain),
```

Comment:
These are placeholders for the actual images that I want to use.  Can you instruct on how to add images to flutter_gallery_assets?